### ISSUE TITLE
Allow modification via unlocking measurement report when set to readonly

### ISSUE THREAD

**Issue by che85** [QIICR/QuantitativeReporting#125]
Users should be allowed to unlock a readonly measurement report. Of the following things need to be taken care:

- [ ] don't touch the original measurement report or segmentation
  - [ ] create a copy of the table node
  - [ ] create a copy of the segmentation node and reference the copy in the new table node
- [ ] user can always go back an force via selecting original/copy in measurement selector

**Comment by che85** [comment]
@fedorov Did I forget something? Feel free to add bullet points

**Comment by che85** [comment]
We might need to handle it differently based on the completion status, where continuing a just "saved" report can be continued and a "completed" cannot.

**Comment by fedorov** [comment]
I am not sure it makes sense to create a copy of the measurements table, since we may not be able to recompute the original measurements. I think to me it would be more intuitive if on report modification the table is reset, user is allowed to modify segmentation, which would then be saved into a new DICOM object, and measurements table is recomputed.

**Comment by che85** [comment]
@fedorov Shall we close this issue or do we want to provide this functionality?

**Comment by fedorov** [comment]
I think it would be good to provide this functionality, but it is not a high priority.

**Comment by che85** [comment]
@fedorov I don't know if this feature is required. It was implemented in https://github.com/che85/Reporting/commit/d88717f85e29c65ef5075a56229251f1315cdc46 but never merged into master

**Comment by fedorov** [comment]
Yes, I am not sure there was a PR. Let's not worry about it for now - let's resolve bugs and features that are needed for @chribaue use cases, and switch to PI-RADS.

**Comment by che85** [comment]
Ok great. I just wanted to go through all issues and close those ones that have been resolved.
